src/services/apiFavService.ts

```diff
@@ -1,69 +1,48 @@
-import { ErrorResponse } from '@/types/ErrorResponse'
-import { handleResponse } from '@/utils/handleResponse'
-import { ServerError } from './authService'
+import { AxiosResponse } from 'axios'
 import { FavResponse, IFavPushItems } from '@/types/Fav'
 import { IProduct } from '@/types/Products'
+import { api } from './apiConfig/apiConfig'
 
 export async function mergeFavs(
   token: string,
   requestItems: IFavPushItems,
 ): Promise<FavResponse> {
   try {
-    const response = await fetch(
-      `${process.env.NEXT_PUBLIC_API_HOST_REMOTE}/favorites`,
-      {
-        method: 'POST',
-        headers: {
-          'Authorization': `Bearer ${token}`,
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(requestItems),
-      },
+    const response: AxiosResponse<FavResponse> = await api.post(
+      '/favorites',
+      requestItems,
     )
 
-    return handleResponse<FavResponse, ErrorResponse>(response)
+    return response.data
```

Comment:
What if `response.status` is not `ok`, will it be handled? Should we consider setting up a global handler using [`axios.interceptors.response.use`](https://stackoverflow.com/a/44351414/8165307)?

Reply:
@DmitriyUshkvarok, could you please clarify that?